recipe/build.sh

```diff
@@ -392,6 +396,17 @@ Mingw_w64_makefiles() {
 Darwin() {
     unset JAVA_HOME
 
+    if [[ $CONDA_BUILD_CROSS_COMPILATION == 1 && $target_platform == osx-arm64 ]]; then
```

Comment:
```suggestion
    if [[ $CONDA_BUILD_CROSS_COMPILATION == 1 ]]; then
```

Reply:
Can you also move this block out of `Darwin()` function and run it for both `linux` and `osx` (not `win`)?

Reply:
Do I create a function to do that? Or copy and paste it twice?

Reply:
You can do it at the beginning of this file or create a function